quiche/src/recovery/gcongestion/bbr2.rs

```diff
@@ -168,7 +169,7 @@ struct Params {
     bw_lo_mode: BwLoMode,
 }
 
-const PARAMS: Params = Params {
+const DEFAULT_PARAMS: Params = Params {
```

Comment:
Any way to make these params private?  It seems like PARAMS was accessed within this crate before your refactor.

Reply:
They are already private to the bbr2 module. Passing a single instance is a good practice we have to enforce.